docs/how-to-guides/rook-ceph-storage.md

```diff
@@ -0,0 +1,293 @@
+
+#### Deploy the component
+
+Execute the following command to deploy the `rook` component:
+
+```bash
+lokoctl component apply rook
+```
+
+Verify the operator pod in the `rook` namespace is in the `Running` state (this may take a few minutes):
+
+```console
+$ kubectl -n rook get pods -l app=rook-ceph-operator
+NAME                                  READY   STATUS    RESTARTS   AGE
+rook-ceph-operator-76d8687f95-6knf8   1/1     Running   0          2m
+```
+
+## Deploy `rook-ceph`
+
+#### Config
+
+Add following contents to the previously created file `storage.lokocfg`:
+
+```tf
+component "rook-ceph" {
+  monitor_count = 3
+
+  node_affinity {
+    key      = "storage.lokomotive.io"
+    operator = "Exists"
+  }
+
+  toleration {
+    key      = "storage.lokomotive.io"
+    operator = "Equal"
+    value    = "ceph"
+    effect   = "NoSchedule"
+  }
+
+  storage_class {
+    enable = true
+  }
+}
+```
+
+- `monitor_count` should be an odd number greater than three and not higher than the `count` variable of workers in the `worker_pool`.
+- `node_affinity` should match the `labels` variable provided in the `worker_pool`.
+- `toleration` should match the `taints` variable provided in the `worker_pool`.
+
+For information about all the available configuration options for the `rook-ceph` component, visit the component's [configuration reference](../configuration-reference/components/rook-ceph.md).
+
+#### Deploy the component
+
+Execute the following command to deploy the `rook-ceph` component:
+
+```bash
+lokoctl component apply rook-ceph
+```
+
+Verify the [OSD](https://docs.ceph.com/docs/master/glossary/#term-ceph-osd-daemon) pods in the `rook` namespace are in the `Running` state (this may take a few minutes):
+
+```console
+$ kubectl -n rook get pods -l app=rook-ceph-osd
+NAME                               READY   STATUS    RESTARTS   AGE
+rook-ceph-osd-0-6d4f69dbf9-26kzl   1/1     Running   0          15m
+rook-ceph-osd-1-86c9597b84-lmh94   1/1     Running   0          15m
+rook-ceph-osd-2-6d97697897-7bprl   1/1     Running   0          15m
+rook-ceph-osd-3-5bfb9d86b-rk6v4    1/1     Running   0          15m
+rook-ceph-osd-4-5b76cb9675-cxkdw   1/1     Running   0          15m
+rook-ceph-osd-5-8c86f5c6c-6qxtz    1/1     Running   0          15m
+rook-ceph-osd-6-5b9cc479b7-vjc9v   1/1     Running   0          15m
+rook-ceph-osd-7-7b84d6cc48-b46z9   1/1     Running   0          15m
+rook-ceph-osd-8-5868969f97-2bn9r   1/1     Running   0          15m
+```
+
+## Access the Ceph dashboard
+
+Ceph dashboard provides valuable visual information. It is an essential tool to monitor the Ceph cluster. Here are steps on how to access it.
+
+Obtain the password for the `admin` Ceph user by running the following command:
+
+```bash
+kubectl -n rook get secret rook-ceph-dashboard-password -o jsonpath="{['data']['password']}" | base64 --decode && echo
+```
+
+Execute the following command to forward port `8443` locally to the Ceph manager pod:
```

Comment:
It seems we could have an option to expose the dashboard using ingress.

Reply:
This is not supported by the chart in upstream. But it can be added.

Reply:
Let's create an issue about it

Reply:
Created https://github.com/kinvolk/lokomotive/issues/759